### ISSUE TITLE
Cannot run Windows nightly build

### ISSUE THREAD

**Issue by Apis035** [IsaacShelton/Adept3x#26]
It complains about missing dll files: `libwinpthread-1`, `libstdc++-6`, `zlib1`, `libzstd`

Also having Windows build packaged with tar.gz seems kind of weird, usually files for Windows is packaged with zip.

**Comment by ghost** [comment]
Yes, Windows package should be ZIP or 7Z.

**Comment by ghost** [comment]
Copy and paste this `adept.exe` into an Adept 2.x installation maybe?

**Comment by IsaacShelton** [comment]
Yes the nightly release is only for getting CI/CD up and working right now. It is not usable yet. Perhaps this could be clarified on the releases page.

It is made intentionally difficult to use right now in order to discourage large usage of it, as many major parts are still being designed and are everything is subject to change.

Many parts of the design are interdependent on each other in nature, so small changes can have big ramifications, and figuring out how to make everything coherent scales in difficulty as you add features. So it's important to try to get as much right as possible before committing to a plan that will address issues that commonly come up later in the lifespans of projects, such as:

- Generics
- Allocators
- Async
- Error Handling
- Traits / Interfaces
- SIMD
- Compile-Time Evaluation
- Contexts
- Exceptions / Effects / Unwinding / Exception-Safety etc.
- Dynamic Linking
- WASM support

just to name a few.

Missing DLL files will also be fixed soon for windows. They are a product of not linking statically with mingw.
